### PR TITLE
fix(offchain): fixed timestamp in graphql

### PR DESCRIPTION
**Pull request by gligneul**
The timestamp was intepreted in milliseconds, but it should be interpreted in seconds.
Also removes the chrono dependency because it is not necessary.

### CODE REVIEW

**Comment by gligneul** [comment]
Tested with examples and it fixes the issue:
```
{
  "data": {
    "inputs": {
      "edges": [
        {
          "node": {
            "timestamp": 1693868070
          }
        }
      ]
    }
  }
}
```

File: offchain/indexer/src/conversions.rs [hunk 1]
```diff
@@ -14,18 +14,7 @@ use rollups_events::{
...
-        Some(timestamp) => timestamp,
-        None => {
-            tracing::warn!(
-                input.metadata.timestamp,
-                "failed to parse timestamp"
-            );
-            Default::default()
-        }
-    };
+    let timestamp = UNIX_EPOCH + Duration::from_secs(input.metadata.timestamp);
```
Comment (tuler) [inline offchain/indexer/src/conversions.rs:17]: Why not only assign input.metadata.timestamp?

Comment (gligneul) [reply offchain/indexer/src/conversions.rs:17]: In the database schema, we are using the timestamp type to represent the input timestamp. In Diesel (the ORM that we use), this type needs to be [`SystemTime`](https://doc.rust-lang.org/std/time/struct.SystemTime.html#) from the standard lib or `NaiveDateTime` from chrono. This operation converts from an integer to `SystemTime`.

> and using UNIX_EPOCH + duration can be used to create a SystemTime instance to represent another fixed point in time.

https://doc.rust-lang.org/std/time/struct.SystemTime.html#associatedconstant.UNIX_EPOCH

Comment (gligneul) [reply offchain/indexer/src/conversions.rs:17]: Here is the list of compatible Rust types for the SQL timestamp: https://docs.diesel.rs/2.0.x/diesel/sql_types/struct.Timestamp.html

**Review by tuler — APPROVED** [review]

**Pull request merged**